NUXT/plugins/languages/french-fr.js

```diff
@@ -84,9 +84,10 @@ module.exports = {
     langsetup: "Commençons par sélectionner une langue !",
     featuresetup: "Sélectionnons des fonctionnalités !",
     enableryd:
-      "Activer Return YouTube Dislike (Retourner les Dislikes de YouTube)",
```

Comment:
I think the [https://returnyoutubedislike.com](https://returnyoutubedislike.com) name should stay for attribution purposes.

Reply:
I'll just fix that on my own after merge, thanks for the changes